recipes/avahi/system/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SysConfigAvahiSystemConan(ConanFile):
+    name = "avahi"
+    version = "system"
+    description = "Conan package for Avahi - Service Discovery for Linux using mDNS/DNS-SD - compatible with Bonjour"
+    topics = ("Avahi", "Bonjour", "DNS-SD", "mDNS")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lathiat/avahi"
+    license = "LGPL-2.1-only"
+    settings = "os"
+
+    def validate(self):
+        if self.settings.os != "Linux":
```

Comment:
To avoid cross building it on Windows to Linux.
@SpaceIm Is this correct?
```suggestion
        if self.settings.os != "Linux" or tools.cross_building(self):
```